cmd/minishift/cmd/oc_env.go

```diff
@@ -42,12 +44,19 @@ type OcShellConfig struct {
 }
 
 func getOcShellConfig(ocPath, forcedShell string) (*OcShellConfig, error) {
+	var cmdLine string
+
 	userShell, err := shell.GetShell(forcedShell)
 	if err != nil {
 		return nil, err
 	}
 
-	cmdLine := "minishift oc-env"
+	if constants.IsProfileExists() {
+		cmdLine = "minishift oc-env --profile " + constants.GetMachineName()
```

Comment:
why do we need the switch here? Is having just a single profile not equivalent with having a profile with the name 'default'?